metadata/containers.go

```diff
@@ -195,6 +195,10 @@ func (s *containerStore) Update(ctx context.Context, container containers.Contai
 		return containers.Container{}, errors.Wrap(err, "update failed validation")
 	}
 
+	if container.Extensions != nil {
```

Comment:
Add `"extensions"`  to the slice on line `149` and move this into the switch at line `184` under `case "extensions"`

Reply:
Does anything need to be done for the protobuf fieldpath thing?

Reply:
That fieldpath generator is only used is for events filtering, since this is an any, that would require having it match against the URL to know how to drill down into the data.

Probably would need some kind of plugin for that, so for the moment I'd say we can ignore it.

Reply:
Ok cool, I've updated this.